contracts/Vault.vy

```diff
@@ -142,57 +203,162 @@ def setSymbol(_symbol: String[20]):
+    @param _rewards The address to use for collecting rewards.
+    """
     assert msg.sender == self.governance
     self.rewards = _rewards
 
 
 @external
 def setDepositLimit(_limit: uint256):
+    """
+    @notice
+        Changes the maximum amount of tokens a single address may have
+        deposited in this Vault.
+
+        Note, this is not how much may be deposited by a single depositor,
+        but the maximum amount that may be deposited across all depositors.
+
+        This may only be called by governance.
+    @param _limit The new deposit limit to use.
+    """
     assert msg.sender == self.governance
     self.depositLimit = _limit
 
 
 @external
 def setPerformanceFee(_fee: uint256):
+    """
+    @notice
+        Used to change the value of `performanceFee`.
+
+        This may only be called by governance.
+    @param _fee The new performance fee to use.
+    """
     assert msg.sender == self.governance
     self.performanceFee = _fee
 
 
 @external
 def setManagementFee(_fee: uint256):
+    """
+    @notice
+        Used to change the value of `managementFee`.
+
+        This may only be called by governance.
+    @param _fee The new management fee to use.
+    """
     assert msg.sender == self.governance
     self.managementFee = _fee
 
 
 @external
 def setGuardian(_guardian: address):
+    """
+    @notice
+        Used to change the address of `guardian`.
+
+        This may only be called by governance or the existing guardian.
+    @param _guardian The new guardian address to use.
+    """
     assert msg.sender in [self.guardian, self.governance]
     self.guardian = _guardian
 
 
 @external
 def setEmergencyShutdown(_active: bool):
     """
-    Activates Vault mode where all Strategies go into full withdrawal
+    @notice
+        Activates or deactivates Vault mode where all Strategies go into full
+        withdrawal.
+
+        During Emergency Shutdown:
+        1. No Users may deposit into the Vault (but may withdrawal as usual.)
+        2. Governance may not add new Strategies.
+        3. Each Strategy must pay back their debt as quickly as reasonable to
+            minimally affect their position.
+        4. Only Governance may undo Emergency Shutdown.
+
+        See contract level note for further details.
+
+        This may only be called by governance or the guardian.
+    @param _active
+        If true, the Vault goes into Emergency Shutdown. If false, the Vault
+        goes back into Normal Operation.
     """
     assert msg.sender in [self.guardian, self.governance]
     self.emergencyShutdown = _active
 
 
 @external
 def setWithdrawalQueue(_queue: address[MAXIMUM_STRATEGIES]):
+    """
+    @notice
+        Updates the withdrawalQueue to match the addresses and order specified
+        by `_queue`.
+
+        There can be fewer strategies than the maximum, as well as fewer than
+        the total number of strategies active with the Vault. `withdrawalQueue`
```

Comment:
`total number of strategies active _**with**_ `  should be `**in** the Vault`

Reply:
es good

Reply:
```suggestion
        the total number of strategies active in the Vault. `withdrawalQueue`
```